LoggingKata.Test/TacoParserTests.cs

```diff
@@ -5,25 +5,32 @@ namespace LoggingKata.Test
 {
     public class TacoParserTests
     {
-        [Fact]
-        public void ShouldDoSomething()
+        [Theory]
+        [InlineData("11.22, 11.33,BLAH")]
+        public void ShouldParse(string str)
         {
-            // TODO: Complete Something, if anything
+            TacoBell blah = TacoParser.ParseLine(str);
+            Assert.Equal(blah.Location, new Point(11.22, 11.33));
+            Assert.Equal(blah.Name, str.Split(',')[2]);
         }
-
         [Theory]
-        [InlineData("Example")]
-        public void ShouldParse(string str)
+        [InlineData("-11.22, -11.33,BLAH")]

```

Comment:
There are probably some more test cases to consider for the parsing method.  Consider identifying a few more `InlineData()` lines that could help verify that the parsing is being done.